frontend/src/views/Lobby.tsx

```diff
@@ -265,13 +296,33 @@ function LobbyPage() {
         <DifficultyButton
           onClick={() => updateDifficultySetting(key)}
           active={difficulty === Difficulty[key as keyof typeof Difficulty]}
-          enabled={currentUser?.nickname === host?.nickname}
+          enabled={isHost(currentUser)}
           title={currentUser?.nickname !== host?.nickname
             ? 'Only the host can change these settings' : undefined}
         >
           {key}
         </DifficultyButton>
       ))}
+
+      <MediumText>Duration</MediumText>
+      <Text>
+        {isHost(currentUser) ? 'Choose a game duration between 1-60 minutes:'
+          : 'The game will last for the following minutes:'}
+      </Text>
+      <NumberInput
+        min={1}
+        max={60}
+        value={duration}
+        disabled={!isHost(currentUser)}
+        onChange={(e) => {
+          const newDuration = Number(e.target.value);
+          if (newDuration >= 0 && newDuration <= 60) {
```

Comment:
The instructions say between 1 and 60 minutes, but you allow any duration between 0 and 60 minutes here. Is there a reason for this discrepancy or was this purposeful? (I saw in the example screencast that you chose 0.1 minutes, for example.)

Also, are we allowing any unit of time, like "30.5910239" or something random as a valid time?

Reply:
Good catch - I'm okay with doing any of those options (1. allowing between (0, 60], 2. allowing between [1, 60], as well as a slightly unrelated 3. limiting to only whole numbers) - what are your preferences?

Reply:
Also, if we go with 3, should we also maybe change the backend to use minutes rather than seconds?

Reply:
I'm OK with any of that really -- I think I might lean towards 2 and 3; however, I'm still thinking that we keep the backend in seconds, as it'd still be nice to trigger different notifications and powerups on a second-by-second basis.

Reply:
Sounds good, implemented